src/drive/mobile/actions/settings.js

```diff
@@ -26,7 +26,7 @@ export const setAnalytics = (analytics, source = 'settings') => (
 ) => {
   dispatch({ type: SET_ANALYTICS, analytics })
   const state = getState()
-  configureReporter(analytics)
+  configureReporter()
```

Comment:
I think this line can be completely removed. The reporter's configuration never changes now, and it is initialized elsewhere, so this line is now useless as far as I can tell.